add info on hiding deleted branches in IDE (#4777)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

2. **Save button &mdash;** The editor has a **Save** button that saves editable files. Pressing the button or using the Command-S or Control-S shortcut saves the file contents. You don't need to save to preview code results in the Console section, but it's necessary before changes appear in a dbt invocation. The File Editor tab shows a blue icon for unsaved changes.

3. **Version Control &mdash;** This menu contains all git-related elements, including the Git actions button. The button updates relevant actions based on your editor's state, such as prompting to pull remote changes, commit and sync when reverted commit changes are present, or creating a merge/pull request when appropriate.

>    - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, and change branches. Keep in mind, you can't delete branches from the remote branches directly using this menu. However, if you [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus:~:text=change%20their%20branch.-,IDE%20Options%20menu,-%E2%80%94%20The%20IDE), it will hide deleted branches from the **Change Branch** list.

Author: ```suggestion
   - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, and change branches. 
      - Keep in mind, although you can't delete branches from the remote branches directly using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus:~:text=change%20their%20branch.-,IDE%20Options%20menu,-%E2%80%94%20The%20IDE). Doing this will hide deleted branches from the **Change Branch** list.
```

2. **Save button &mdash;** The editor has a **Save** button that saves editable files. Pressing the button or using the Command-S or Control-S shortcut saves the file contents. You don't need to save to preview code results in the Console section, but it's necessary before changes appear in a dbt invocation. The File Editor tab shows a blue icon for unsaved changes.

3. **Version Control &mdash;** This menu contains all git-related elements, including the Git actions button. The button updates relevant actions based on your editor's state, such as prompting to pull remote changes, commit and sync when reverted commit changes are present, or creating a merge/pull request when appropriate.
   - The dropdown menu on the Git actions button allows users to revert changes, refresh Git state, create merge/pull requests, and change branches.

>      - Keep in mind that although you can't delete local branches in the IDE using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus), which will delete all local branches and resync with all existing remote branches.

Author: ```suggestion
     - Keep in mind that although you can't delete local branches in the IDE using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus), which deletes your local branches and refreshes with the current remote branches, effectively removing the deleted ones.
```

>      - Keep in mind, although you can't delete branches from the remote branches directly using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus). Doing this will hide deleted branches from the **Change Branch** list.

Reviewer: it's not really `hiding` them. recloning will delete the branches locally and then get all the remote branches which no longer has the ones you have deleted.

perhaps:
```
Although you can't delete local branches in the IDE using this menu, you can reclone your repository which will delete all local branches and resync with all existing remote branches.
```

>      - Keep in mind, although you can't delete branches from the remote branches directly using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus). Doing this will hide deleted branches from the **Change Branch** list.

Author: ```suggestion
     - Keep in mind that although you can't delete local branches in the IDE using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus), which will delete all local branches and resync with all existing remote branches.
```

>      - Keep in mind, although you can't delete branches from the remote branches directly using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus:~:text=change%20their%20branch.-,IDE%20Options%20menu,-%E2%80%94%20The%20IDE). Doing this will hide deleted branches from the **Change Branch** list.

Author: ```suggestion
     - Keep in mind, although you can't delete branches from the remote branches directly using this menu, you can [reclone your repository](/docs/cloud/dbt-cloud-ide/ide-user-interface#modals-and-menus). Doing this will hide deleted branches from the **Change Branch** list.
```
